src/devices/tuya.ts

```diff
@@ -2163,6 +2234,18 @@ const definitions: Definition[] = [
             await reporting.bind(device.getEndpoint(1), coordinatorEndpoint, ['genOnOff']);
         },
     },
+    {
+        fingerprint: tuya.fingerprint('TS0001', ['_TZ3000_veu2v775']),
+        model: 'TS0001_switch_module_2',
+        vendor: 'TuYa',
+        description: '1 gang mini switch module',
+        whiteLabel: [{vendor: 'MOES', model: 'ZM-104-M'}],
```

Comment:
Could you move these and the device below to `moes.js` and remove the whiteLabel from it? (update the vendor/model accordingly)